dbt/task/compile.py

```diff
@@ -6,6 +6,8 @@
 import dbt.project
 from collections import defaultdict
 
+from ..compilation import Linker
```

Comment:
we're importing `project` using `import dbt.project`.  Can't we use `import dbt.compilation` here?  If we can, I think we should for consistency